.golangci.next.reference.yml

```
@@ -261,6 +261,78 @@ linters:
   # Default: false
   fast: true
 
+  # Defines a set of rules to ignore issues.
+  # It does not skip the analysis, and so don't ignore "typecheck" errors.
+  exclusions:
+    # Mode of the generated files analysis.
+    #
+    # - `strict`: sources are excluded by following strictly the Go generated file convention.
+    #    Source files that have lines matching only the following regular expression will be excluded: `^// Code generated .* DO NOT EDIT\.$`
+    #    This line must appear before the first non-comment, non-blank text in the file.
+    #    https://go.dev/s/generatedcode
+    # - `lax`: sources are excluded if they contain lines `autogenerated file`, `code generated`, `do not edit`, etc.
+    # - `disable`: disable the generated files exclusion.
+    #
+    # Default: lax
+    generated: strict
+    # Log a warning if an exclusion rule is unused.
+    # Default: false
+    warn-unused: true
+    # Predefined exclusion rules.
+    # Default: []
+    presets:
+      - comments
+      - stdErrorHandling
+      - commonFalsePositives
+      - legacy
```

Maybe better is to use `-` as a delimiter between words instead of camelCase? 
```suggestion
      - comments
      - std-error-handling
      - common-false-positives
      - legacy
```

Or make preset names shorter with an extended description.

---

I prefer to use camelCase because these are not field names.

---

I will change the values but we agree those values have been documented inside the proposal and introduced by another PR. This PR is just adding then to the documentation.